config/packages/doctrine_migrations.yaml

```
@@ -2,5 +2,5 @@ doctrine_migrations:
     migrations_paths:
         # namespace is arbitrary but should be different from App\Migrations
         # as migrations classes should NOT be autoloaded
-        'DoctrineMigrations': '%kernel.project_dir%/migrations'
+        'DoctrineMigrations': '%kernel.project_dir%/src/Infrastructure/Persistence/Doctrine/Migrations'
```

```suggestion
       migrations_paths:
        App\Infrastructure\Persistence\Doctrine\Migrations: '%kernel.project_dir%/src/Infrastructure/Persistence/Doctrine/Migrations'
```

Le fait d'écrire la config de cette manière va permettre de te créer tes migrations avec le bon namespace.

---

Du coup c'est quoi la stratégie après avoir changé la config dans `doctrine_migrations` ? Delete toutes les migrations existantes et relancer `make dbmigration`, et reset ma db en local ? Ou bien corriger les namespaces dans les migrations existantes ?

---

Tu peux compléter celles existantes je pense. Pas besoin de tout rebuildé.